man/systemd.netdev.xml

```diff
@@ -342,6 +342,24 @@
           </listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>GroupForwardMask=</varname></term>
+          <listitem>
+            <para>Bitmask to allow forwarding of link local frames with address
+            01-80-C2-00-00-0X on a bridge device. Only values that set bits
```

Comment:
"ip link" and "ifconfig" both format MAC addresses using ":" as byte separators, and it appears to me this is the most common way to format them, hence let's please do that here too.